### ISSUE TITLE
NearestNeighbors thumbnail upscale

### ISSUE THREAD

**Issue by Satellence** [files-community/Files#14346]
### What feature or improvement do you think would benefit Files?

When working with small images, zooming in on them shows up as a highly blurred something. I work with pixel art pictures, and have been looking for a solution that would allow me to preview pictures comfortably for a long time. Being able to enable displaying thumbnails without anti-aliasing filters at upscale would help a lot.

### Requirements

I don't know what to add besides the above

### Files Version

3.0.15.0

### Windows Version

10.0.22621.2861

**Comment by yaira2** [comment]
Are you talking about the preview pane, or do you mean in the layout itself?

**Comment by Satellence** [comment]
I'm not sure I understood the question correctly. I'm talking about displaying file thumbnails in Explorer 
![image](https://github.com/files-community/Files/assets/1390832/4ba474a7-37fc-4c82-8355-a964d1258fe9)

**Comment by yaira2** [comment]
Does refreshing the view show a higher res thumbnail?

**Comment by yaira2** [comment]
We're also tracking similar feedback in #10067

**Comment by Satellence** [comment]
> Does refreshing the view show a higher res thumbnail?
No. And it shouldn't, these files are very low resolution (32x32 pixels). I just wanted to suggest an optional setting to make preview images magnified without anti-aliasing filters

**Comment by yaira2** [comment]
What do they look like in File Explorer?

**Comment by yaira2** [comment]
The way you would use this feature, would it be for all folders, or would it only be for specific locations?

**Comment by Satellence** [comment]
> The way you would use this feature, would it be for all folders, or would it only be for specific locations?

I'm always in favor of finer customization, which is why I'd like to specify separately for folders. For photos and other images, this method of scaling images does not work very well.

**Comment by felix-x1** [comment]
Is there, as of now, any way to fix blurry thumbnails on low resolution images?
Most other file browsers manage to keep even low res images looking sharp, is it possible to do the same here?

**Comment by vukkashin** [comment]
I would love to have this option as well! I work on a lot of small sprites and would like to be able to see them without them being blurry.
